src/layers/SchematicLayer.ts

```diff
@@ -101,6 +109,15 @@ const foldInterlacedRenderObjects =
     }
   };
 
+// Perforation
+const perforationColors = {
+  stroke: 'rgba(0, 0, 0, 0.25)',
+  yellow: '#FFFC00',
+  grey: 'gray',
+  red: '#FF5050',
+  transparent: 'rgba(255, 255, 255, 0)',
+};
```

Comment:
Can you create a `PerforationOptions` in the [`Schematic interfaces file`](https://github.com/equinor/esv-intersection/blob/master/src/layers/schematicInterfaces.ts#L135) for these options?
We have some examples of setting the default properties as well there.